pkg/apis/cluster/v1alpha1/machine.go

```diff
@@ -43,8 +43,10 @@ type CustomMachine struct {
 
 // CustomMachineSpec defines kubernetes cluster master and nodes.
 type CustomMachineSpec struct {
-	Master []Machine `json:"master,omitempty"`
-	Nodes  []Machine `json:"node,omitempty"`
+	// MachineOwner  is the reference of kurator cluster which associate with this CustomMachine
+	MachineOwner *corev1.ObjectReference `json:"machineRef,omitempty"`
```

Comment:
Not sure i understand this field, who is the owner

Reply:
customeCluster. I want find customeCluster via customMachine, and not sure current "MachineOwner" field is a suitable way of realizing this.